File: app/src/main/java/vn/loitp/app/activity/customviews/textview/verticalmarqueetextview/VerticalMarqueeTextViewActivity.java
package vn.loitp.app.activity.customviews.textview.verticalmarqueetextview;

import android.os.Bundle;

import com.core.base.BaseFontActivity;

import loitp.basemaster.R;

public class VerticalMarqueeTextViewActivity extends BaseFontActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_vertical_marque_textview;
    }

}

File: app/src/main/java/vn/loitp/app/activity/customviews/textview/verticalmarqueetextview/VerticalMarqueeTextViewActivity.kt
package vn.loitp.app.activity.customviews.textview.verticalmarqueetextview

import com.annotation.IsFullScreen
import com.annotation.LogTag
import com.core.base.BaseFontActivity
import vn.loitp.app.R

@LogTag("VerticalMarqueeTextViewActivity")
@IsFullScreen(false)
class VerticalMarqueeTextViewActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_text_view_vertical_marque
    }
}
